variables.tf

```diff
@@ -23,8 +24,8 @@ variable "regional" {
 
 variable "region" {
   type        = string
-  description = "The region to host the cluster in"
-  default     = null
+  description = "The region to host the cluster in. Default: us-central1"
+  default     = "us-central1"
 }
 
 variable "zones" {
```

Comment:
Is this variable a zone or an array of zones?
The description says the only zone

Reply:
Done, I used the same approach to determine the zones to use than the official module